source/Generic/PlayState/ViewModels/PlayStateManagerViewModel.cs

```diff
@@ -415,12 +424,23 @@ public bool SwitchGameState(PlayStateData gameData)
                         notificationType = processesSuspended ? NotificationTypes.Resumed : NotificationTypes.PlaytimeResumed;
                         gameData.Stopwatch.Stop();
                         logger.Debug($"Game {gameData.Game.Name} resumed in mode {gameData.SuspendMode}");
+                        if (reminder)
+                        {
+                            gameData.ReminderTimer.Stop();
+                        }
                     }
                     else
                     {
                         gameData.IsSuspended = true;
                         notificationType = processesSuspended ? NotificationTypes.Suspended : NotificationTypes.PlaytimeSuspended;
                         gameData.Stopwatch.Start();
+                        if (reminder)
+                        {
+                            gameData.ReminderTimer = new DispatcherTimer();
+                            gameData.ReminderTimer.Interval = TimeSpan.FromMinutes(1);
+                            gameData.ReminderTimer.Tick += (sender, e) => ShowNotificationReminderIfCurrentGameIsSuspended(gameData);       
+                            gameData.ReminderTimer.Start();
```

Comment:
I don't like that the timer is being created here, outside of the object itself. I think there are two options:

- Have a single timer in the `PlayStateManager` that checks all Data objects and sends notifications if needed. Easiest to implement.
- Have a timer created inside the `PlayStateData`. It would also need an Event that sends a signal that a notification should be sent. The PlayStateManager subscribes to this event. Remember that events should be unsubscribed when disposing the object

Reply:
@darklinkpower I moved the reminder to PlayStateData, though it still add the method call in the viewmodel in order to make it simpler. Is this approach fine for you?